source/sass/patterns/molecules/inline-list.scss

```diff
@@ -0,0 +1,36 @@
+@import "../../mixins/typography";
+
+.inline-list {
+  @include list-style-none();
+  overflow-y: hidden;
+}
+
+.inline-list__item {
+  display: inline-block;
+  font-family: $font-secondary;
```

Comment:
Probably shouldn't be here. Needed when used in a content header though, so either add a class there, or maybe even set it on the `.content-header` itself?

Reply:
This feels like a parameter; supports your suggestion in https://github.com/libero/pattern-library/pull/111/files/c87b48f5d9d87619c84539cc61d1622dadd65860#r294247291

Reply:
Can move it to `.inline-list` along with the other typography. Unless there're examples of non-`$font-secondary`-styled list I think this should be okay.